modules/mobile/spec/models/authorized_services_spec.rb

```diff
@@ -0,0 +1,406 @@
+# frozen_string_literal: true
```

Comment:
These specs are largely repetitive and I considered created shared examples to reduce the complexity but that wasn't simple and I backed out of it. If I weren't already a little past timebox on this, I'd consider doing so. As it is, the sheer length of the specs made it difficult to proofread.